packages/openapi/src/middleware.test.ts

```diff
@@ -186,7 +186,6 @@ describe('OpenAPI Validator', (): void => {
     test.each`
       status | body                                                                    | message                                                           | description
       ${202} | ${{}}                                                                   | ${'An unknown status code was used and no default was provided.'} | ${'status code'}
-      ${201} | ${{ ...body, invalid: 'field' }}                                        | ${'response must NOT have additional properties: invalid'}        | ${'body with additional property'}
```

Comment:
I expected this since it's no longer restricting additional properties, but I'm surprised this case in a previous test is still working with `unevaluatedProperties: false` (and that this case wouldn't behave the same way)

https://github.com/interledger/open-payments/blob/32e089c431cbb17f93e1a9d5c218e84d0618a593/packages/openapi/src/middleware.test.ts#L133

Reply:
The 
```
${{ additionalProp: 'disallowed' }} | ${'body must NOT have additional properties: additionalProp'} | ${'invalid additional property'}
```

 test looks like its for the incoming-payments _request_ instead?

Reply:
Ah, yes. that's it.